placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -439,4 +443,23 @@ public void setLanguageCode(@Nullable String languageCode) {
         this.languageCode = languageCode;
         api.setLanguageCode(this.languageCode);
     }
+
```

Comment:
The background was blank and didn't have the normal EditText underline, but it seems to be a problem with the demo app not using an AppCompat theme. Fixed that.